src/module/api/timeline/search/index.ts

```diff
@@ -0,0 +1,50 @@
+import apigents from "@venturemark/apigents";
+import * as env from "module/env";
+import * as spec from "./spec";
+
+type DataField = {
+  [key: string]: string;
+};
+
+type ChunkingObject = {
+  perpage: number;
+  pointer: number;
+};
+
+type ApiObject = {
+  chunking: ChunkingObject;
+};
+
+type DataObject = {
+  metadata: DataField;
+};
+
+export interface TimelineSearchI {
+  api?: ApiObject;
+  obj: DataObject[];
+}
+
+export interface TimelineSearchO {
+  api?: ApiObject;
+  obj: DataObject[];
+}
+
+export function Search(request: TimelineSearchI): TimelineSearchO | null {
+  {
+    const client = new apigents.Timeline.Client(env.APIEndpoint());
+    const req = new apigents.Timeline.Search.I();
+
+    req.setObjList(request);
+
+    client.search(req, {}, function (err: any, res: TimelineSearchO) {
```

Comment:
At present I'm receiving a "503 (Service Unavailable)" error:
![React_App](https://user-images.githubusercontent.com/1753324/102959552-b657fb80-4494-11eb-8776-a7c48fa77deb.jpg)

Any idea what this indicates?

Reply:
Fixed via 

- https://github.com/venturemark/flux/pull/1
- https://github.com/venturemark/apiserver/pull/62